Using attrib.exe to Manage States (#482)
owncloud/docs-client-desktop: modules/ROOT/pages/vfs.adoc

. When done, Explorer will show the file in _Placeholder_ state.
+
image::vfs/vfs-setup-successful-explorer.png[Setup Successful Explorer, width=600,pdfwidth=70%]

## Using attrib.exe to Manage States

> For experienced users, xref:vfs-states[VFS states] can be managed also via the command line using `attrib`. This is particular useful if you want to change the state of a complete folder/subfolder. Open a `PowerShell` command line window like via the start menu and change to the directory of the ownCloud sync folder/subfolder that contains the data you want to bulk change.

Reviewer: ```suggestion
Experienced users can also manage xref:vfs-states[VFS states] via the command line using `attrib`. This is particularly useful if you want to change the state of a complete folder/subfolder. Open a `PowerShell` command line window like via the start menu and change to the directory of the ownCloud sync folder/subfolder that contains the data you want to bulk change.
```

> Use the following command for help and to adapt switches according your needs.

Reviewer: ```suggestion
Use the following command for help and to adapt switches according to your needs.
```

[source,powershell]
----
attrib /?
----

* Make a set of files or folders always available including subfolders:
+
[source,powershell]
----
attrib -U +P .\* /D /S
----

> * Make a set of files or folders online only including subfolders:

Reviewer: ```suggestion
* Make a set of files or folders only online available, including subfolders:
```
